### PR TITLE
Vigilant feature marks cryptographically verified signatures (whose GPG key's email has been verified by github) as unverified.

### LINKED ISSUE #8173: Vigilant feature marks cryptographically verified signatures (whose GPG key's email has been verified by github) as unverified.
There is no issue on this repo already wrt. to this technically wrong usage of the (paraphrased) phrase "(cryptographically) unverified".

### What article on docs.github.com is affected?

See diff in https://github.com/github/docs/pull/8172. And below picture.
2021-07-13T17:59:21,043870146+02:00

### What part(s) of the article would you like to see updated?

Read https://github.com/github/docs/pull/8172.

### Additional information

None, but I'll be happy to follow up/answer any questions if I haven't presented the issue in an understandable way.

### PR DESCRIPTION
**Pull request by x10an14**
It says cryptographically unverified, when it in fact _is_ cryptographically verifiable/verified.

I'd suggest Github changes this scenario to `partially verified` at the very least (as opposed to "WARNING! WARNING! UNVERIFIED!", but ideally I'd like to be allowed to set whatever author email I want in my git commits, without that impeding on the veracity of my GPG signatures.

My use-case is to not spread an email I care about more than I have to. Enough trawlers already - few if any care to parse GPG keys and extract (it seems).
9/10 times (in my personal experience) they just trawl github commit logs with a webscraping service instead.

Those who think that the emails must match (that the git commit/tag author email gives any security at all) have a nasty surprise-in-waiting.

### Why:

Closes https://github.com/github/docs/issues/8173.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by rpshbhandari — CHANGES_REQUESTED** [review]
the claim is valid

**Comment by ramyaparimi** [comment]
@x10an14 Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by x10an14** [comment]
> the claim is valid

? Are you referring to the gpg signature?

**Review by Roth386 — DISMISSED** [review]

File: content/github/authenticating-to-github/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ redirect_from:
...
 
-By default commits and tags are marked "Verified" if they are signed with a GPG or S/MIME key that was successfully verified. If a commit or tag has a signature that can't be verified, {% data variables.product.prodname_dotcom %} marks the commit or tag "Unverified." In all other cases no verification status is displayed.
+By default commits and tags are marked "Verified" if they are signed with a GPG or S/MIME key that was successfully verified. If a commit or tag has a signature that can't be verified - or can be cryptographically verified but the GPG key's email doesn't match the commit's author email - then {% data variables.product.prodname_dotcom %} marks the commit or tag "Unverified." In all other cases no verification status is displayed.
```
Comment (lecoursen) [inline content/github/authenticating-to-github/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits.md:19]: Hi @x10an14! When we refer to "verified" in this article, we mean verified by GitHub, which requires the matching email address... not cryptographically verified.

```suggestion
By default commits and tags are marked "Verified" if they are signed with a GPG or S/MIME key that was successfully verified. If a commit or tag has a signature that can't be verified by {% data variables.product.prodname_dotcom %}, we mark the commit or tag "Unverified." In all other cases no verification status is displayed.
```

File: content/github/authenticating-to-github/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits.md [hunk 2]
```diff
@@ -16,15 +16,15 @@ redirect_from:
...
 
 However, you can give other users increased confidence in the identity attributed to your commits and tags by enabling vigilant mode in your {% data variables.product.prodname_dotcom %} settings. With vigilant mode enabled, all of your commits and tags are marked with one of three verification statuses.
 
 ![Signature verification statuses](/assets/images/help/commits/signature-verification-statuses.png)
 
 {% data reusables.identity-and-permissions.vigilant-mode-verification-statuses %}
 
-You should only enable vigilant mode if you sign all of your commits and tags. After enabling this mode, any unsigned commits or tags that you generate locally and push to {% data variables.product.prodname_dotcom %} will be marked "Unverified."
+You should only enable vigilant mode if you sign all of your commits and tags, and ensure that you _always_ set git commit author email to an (identical) email listed on your GPG primary key when using Github. After enabling this mode, any unsigned commits or tags that you generate locally and push to {% data variables.product.prodname_dotcom %} will be marked "Unverified."
```
Comment (lecoursen) [inline content/github/authenticating-to-github/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits.md:27]: We check the committer address, not the author address.

```suggestion
You should only enable vigilant mode if you sign all of your commits and tags and use an email address that is verified in {% data variables.product.product_name %} for your committer email address. After enabling this mode, any unsigned commits or tags that you generate locally and push to {% data variables.product.prodname_dotcom %} will be marked "Unverified."
```

**Review by lecoursen — COMMENTED** [review]
Thanks @x10an14! I left a few suggested changes, which I'll apply now, then merge! ⚡

File: content/github/authenticating-to-github/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits.md [hunk 3]
```diff
@@ -16,15 +16,15 @@ redirect_from:
...
 
 However, you can give other users increased confidence in the identity attributed to your commits and tags by enabling vigilant mode in your {% data variables.product.prodname_dotcom %} settings. With vigilant mode enabled, all of your commits and tags are marked with one of three verification statuses.
 
 ![Signature verification statuses](/assets/images/help/commits/signature-verification-statuses.png)
 
 {% data reusables.identity-and-permissions.vigilant-mode-verification-statuses %}
 
-You should only enable vigilant mode if you sign all of your commits and tags. After enabling this mode, any unsigned commits or tags that you generate locally and push to {% data variables.product.prodname_dotcom %} will be marked "Unverified."
+You should only enable vigilant mode if you sign all of your commits and tags and use an email address that is verified in {% data variables.product.product_name %} for your committer email address. After enabling this mode, any unsigned commits or tags that you generate locally and push to {% data variables.product.prodname_dotcom %} will be marked "Unverified."
```
Comment (lecoursen) [inline content/github/authenticating-to-github/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits.md:27]:
```suggestion
You should only enable vigilant mode if you sign all of your commits and tags and use an email address that is verified for your account on {% data variables.product.product_name %} as your committer email address. After enabling this mode, any unsigned commits or tags that you generate locally and push to {% data variables.product.prodname_dotcom %} will be marked "Unverified."
```

**Review by lecoursen — APPROVED** [review]

**Comment by x10an14** [comment]
I'm sorry, but I don't think you've solved the core of the issue here. Is this perhaps not the right place to come with this customer feedback?

Users will look at "Vigilant" and see the "verified/unverified" tags - and act/react on a basis if something seems "verified and safe" vs "unverified and unsafe".
And only the fewest users will care to git clone the commits themselves, and double-check the GPG commit signature themselves - when and if your "Vigilant" system marks a commit as "unverified".

In essence, I get the UX that you're hiding/diminishing the value of a "cryptographically verifiable signature" in your efforts to promote the Github's vendor-specific "Vigilant".

My commits (pushed to github) will still be marked as "unverified", and that's doing _me_ a disservice since I've gone through the trouble of learning GPG, handling my keys securely with smart-cards, and signing my commits so people can trust that _this is *me*_.

Not some random rootkit on my machine or random mischievous agent on the internet.

I don't mind Github wanting to promote/make use of "Vigilant", but why does it need to react when these two emails don't match? Given that the cryptographic signature is valid???

**Comment by x10an14** [comment]
Re-reading your edits to the proposed changes, I see that I actually _am_ fulfilling all requirements. Yet my commits get marked "Unverified".

Just see [here](https://github.com/maxjacobson/vim-fzf-coauthorship/pull/2/commits/f1c693d360931a058a8d73140949e77cc9598128).

> By default commits and tags are marked "Verified" if they are signed with a GPG or S/MIME key that was successfully verified. If a commit or tag has a signature that can't be verified by {% data variables.product.prodname_dotcom %}, we mark the commit or tag "Unverified." In all other cases no verification status is displayed.

It is cryptographically verifiable.

> You should only enable vigilant mode if you sign all of your commits and tags and use an email address that is verified for your account on {% data variables.product.product_name %} as your committer email address.

But both my committer (AND GPG email) are verified with you (read: Github) - so why are my commits marked as unverified then?

The only thing I can see (but there might be something I don't see) is that you (for some unfathomable reason to me) compare the committer email with the emails listed in the GPG key.

**Comment by ramyaparimi** [comment]
@x10an14 This is great feedback to give to https://support.github.com/contact/feedback :yellow_heart:

**Pull request merged**